release-notes/6.0/preview/api-diff/preview4/.Net/6.0-preview4_System.md

```diff
@@ -0,0 +1,202 @@
+# System
+
+``` diff
+ namespace System {
+-    public readonly struct Byte : IComparable, IComparable<byte>, IConvertible, IEquatable<byte>, IFormattable
```

Comment:
`ISpanFormattable` addition to core numeric types LGTM.

CC. @pgovind, @stephentoub